library/lib.sh

```diff
@@ -21,6 +21,7 @@ rolesInstallAnsible() {
         PYTHON_VERSION=3.9
         rlRun "dnf install python$PYTHON_VERSION -y"
         rlRun "python$PYTHON_VERSION -m pip install ansible==$ANSIBLE_VER.*"
+        dnf install selinux # needed for delegate_to: localhost for file, copy, etc.
```

Comment:
```suggestion
```

Reply:
In order to install from rpm/dnf/yum, the name of the package is different depending on the platform - for example, see how the selinux role installs the selinux python libraries - https://github.com/linux-system-roles/selinux/blob/main/tasks/ensure_selinux_packages.yml